--- screen_old.jl	2025-04-17 11:40:39.225074690 -0700
+++ screen.jl	2025-04-16 18:51:31.526780786 -0700
@@ -404,7 +404,8 @@
             GLFW.SetWindowAttrib(glw, GLFW.FLOATING, config.float)
         end
         if !isnothing(config.monitor)
-            GLFW.SetWindowMonitor(glw, config.monitor)
+            vd = GLFW.GetVideoMode(config.monitor)
+            GLFW.SetWindowMonitor(glw, config.monitor, 0.0, 0.0, vd.width, vd.height, vd.refreshrate)
         end
     end
 
